styles/app.css

```diff
@@ -0,0 +1,150 @@
+body {
+  margin: 0;
+  font-family: sans-serif;
+}
+
+ul {
+  padding-left: 0;
+  list-style: none;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+h3 {
+  font-weight: normal;
+}
+
+a {
+  text-decoration: none;
+}
+
+.section {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  padding:  0 8rem;
+}
+
+.top-bar {
+  background: #66b5e8;
+  overflow: auto;
+  color: #fefefe;
+  padding: 0 9rem;
+}
+
+.top-bar__title {
+  float: left;
+  font-weight: bolder;
+}
+
+.top-bar__nav {
+  float: right;
+  margin: 1.5rem 0;
+}
```

Comment:
Add negative margin left and right to account for the margin on the `.top-bar__link`